Fix a couple of things with `find-node` recipe (#23)
unifiedjs/unifiedjs.github.io: doc/learn/find-node.md

### What is a node

A node is a single language specific unit inside a [syntax tree][syntax-tree].

> For example: a heading in markdown, or anchor element in HTML. In unified, nodes

Reviewer: ```suggestion
For example: a heading in markdown, or anchor element in HTML.  In unified, nodes
```

[tree traversal][tree-traversal] of a [syntax tree][syntax-tree].

Unified compatible utilities should be used for finding a node.

> Utilities are functions that work with nodes. All specifications

Reviewer: ```suggestion
Utilities are functions that work with nodes.  All specifications
```

that extend [unist][] can use the [unist utilities][unist-utils],
but they can also have their own utilities for more specific nodes.

> To start finding nodes for your input you'll need:

Reviewer: ```suggestion
To start finding nodes for your input you’ll need:
```

*   A processor (such as [`remark`][remark]).
*   A utility of choice.

For this example we use [`remark`][remark]

> and [`unist-util-find`][unist-util-find]. We want to find

Reviewer: ```suggestion
and [`unist-util-find`][unist-util-find].  We want to find
```
